sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/system/SystemStatusListener.java

```diff
@@ -27,9 +27,25 @@
 
 /**
  * @author jialiang.linjl
+ * @author <a href="mailto:doob.yi@qq.com">doob</a>
  */
 public class SystemStatusListener implements Runnable {
 
+    public static final String CALCULATE_BY_SINGLE_CORE_KEY =
+            "csp.sentinel.system.status.cpuUsageCalculateBySingleCore";
```

Comment:
I suggest that we could do a lazy initialization in, for example, `run()`?
It will be more friendly for various implementations.

Reply:
> I suggest that we could do a lazy initialization in, for example, `run()`?
> It will be more friendly for various implementations.

good idea

Reply:
> I suggest that we could do a lazy initialization in, for example, `run()`?
> It will be more friendly for various implementations.

fixed;

i lazy init in `getProcessCpuUsage()`，because i don't think `run()` should care about this parameter

Reply:
> i lazy init in `getProcessCpuUsage()`，because i don't think `run()` should care about this parameter

Though it hasn't been synchronized here I think it's not a big deal.